e2e/common/operator_metrics_test.go

```diff
@@ -81,14 +81,19 @@ func TestMetrics(t *testing.T) {
 				AddStep(MatchFields(IgnoreExtras, Fields{
 					"LoggerName":  Equal("camel-k.controller.build"),
 					"Message":     Equal("Build state transition"),
-					"Phase":       Equal(string(v1.BuildPhasePending)),
+					"Phase":       MatchFields(IgnoreExtras,
```

Comment:
Could `"Phase": Equal(v1.BuildPhasePending)` work?

Reply:
I don't think so. This seemed as one of the things causing this test to fail, since the Phase element on the LogEntry structure was changed (because the logs are not normalized themselves, having either strings integers for that element).